app/src/main/java/com/framgia/englishconversation/screen/setting/SettingViewModel.java

```diff
@@ -3,14 +3,22 @@
 import android.content.Intent;
 import android.databinding.BaseObservable;
 import android.databinding.Bindable;
+import android.support.annotation.NonNull;
+import android.util.Log;
+
 import com.framgia.englishconversation.BR;
 import com.framgia.englishconversation.R;
 import com.framgia.englishconversation.data.model.Setting;
 import com.framgia.englishconversation.data.model.UserModel;
+import com.framgia.englishconversation.screen.changepassword.ChangePasswordFragment;
 import com.framgia.englishconversation.screen.login.LoginActivity;
 import com.framgia.englishconversation.screen.main.MainActivity;
 import com.framgia.englishconversation.utils.navigator.Navigator;
 import com.google.android.gms.common.api.GoogleApiClient;
+import com.google.android.gms.tasks.OnCompleteListener;
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.auth.FirebaseAuth;
```

Comment:
ERROR: Unused import: com.google.firebase.auth.FirebaseAuth. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports